src/VisualStudio/Core/Def/TaskList/ProjectExternalErrorReporter.cs

```diff
@@ -101,16 +101,8 @@ public int AddNewErrors(IVsEnumExternalErrors pErrors)
                     error.bstrText,
                     GetDiagnosticSeverity(error),
                     _language,
-                    mappedFilePath: null,
-                    mappedStartLine: 0,
-                    mappedStartColumn: 0,
-                    mappedEndLine: 0,
-                    mappedEndColumn: 0,
-                    originalFilePath: null,
-                    originalStartLine: 0,
-                    originalStartColumn: 0,
-                    originalEndLine: 0,
-                    originalEndColumn: 0));
+                    mappedSpan: null,
+                    originalSpan: null));
```

Comment:
i heavily dislike this.  it feels like we have diagnostics we have no idea what to do with, so we just throw our hands up in the air.  We shoudl mandate that all diagnostics have an actual location we can take the user to so they can actually fix the thing in question.